Form/DataTransformer/ModelsToArrayTransformer.php

```diff
@@ -107,7 +108,12 @@ public function reverseTransform($keys)
         }
 
         if (count($notFound) > 0) {
-            throw new TransformationFailedException(sprintf('The entities with keys "%s" could not be found', implode('", "', $notFound)));
+            throw new TransformationFailedException(
```

Comment:
Can you split this into multiple commits? One for the reformat and one for the code change?